src/world/Spell/Spell.Legacy.cpp

```diff
@@ -988,10 +987,15 @@ uint8 Spell::prepare(SpellCastTargets* targets)
             //p_caster->setAttackTimer(m_timer + 1000, false);
         }
 
+        // TODO: This is partially correct but partially not
+        // afaik only some aurastates should be removed when casting a specific spell
+        // but I don't think removing the aurastate already in ::prepare is the correct way
+        // Commenting this out for now but it needs more investigation -Appled
+
         // aura state removal
-        if (GetSpellInfo() && GetSpellInfo()->getCasterAuraState() != AURASTATE_NONE && GetSpellInfo()->getCasterAuraState() != AURASTATE_FLAG_JUDGEMENT)
+        /*if (GetSpellInfo() && GetSpellInfo()->getCasterAuraState() != AURASTATE_NONE && GetSpellInfo()->getCasterAuraState() != AURASTATE_FLAG_JUDGEMENT)
```

Comment:
Please do not comment out blocks of code - if the code is not supposed to be there, remove it.